### PR TITLE
Use persistent storage

### PR DESCRIPTION
**Pull request by clayferg**
Making sure everything makes it to the master.

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 1]
```diff
@@ -26,28 +32,33 @@
...
-
-  private static class NewComment {
+  private static class Comment {
+    long id;
+    long timestamp;
     String username;
     String comment;
 
-    public NewComment(String username, String comment) {
+    public Comment(long id, long timestamp, String username, String comment) {
```
Comment (ciddu) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:41]: camelcase  userName

https://google.github.io/styleguide/javaguide.html#s5.3-camel-case

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 2]
```diff
@@ -26,28 +32,33 @@
...
+      this.timestamp = timestamp;
       this.username = username;
       this.comment = comment;
     }
   }
 
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    String username = getStringParameter(request, "username");
+    String comment = getStringParameter(request, "comment");
```
Comment (ciddu) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:52]: what if both these are empty? do you still want to write to the datastore?

**Review by ciddu — CHANGES_REQUESTED** [review]

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java (hunk 1 above)
Comment (clayferg) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: Right, thanks.

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java (hunk 2 above)
Comment (clayferg) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: Good point, I stopped it from writing when comment is empty and made an empty name = anonymous.

**Review by ciddu — APPROVED** [review]

**Pull request merged**